ribs/emitters/_gaussian_emitter.py

```diff
@@ -26,10 +26,10 @@ class GaussianEmitter(EmitterBase):
             distribution when the archive is not empty. Note we assume
             the Gaussian is diagonal, so if this argument is an array, it
             must be 1D.
-        sigma0 (float or array-like): Standard deviation of the Gaussian
-            distribution when the archive is empty. If this argument is
-            None, then it defaults to sigma. Note we assume the Gaussian is
-            diagonal, so if this argument is an array, it must be 1D.
+        initial_solutions (array-like): An (n, solution_dim) array of solutions
```

Comment:
You also need to figure out how to deal with x0, because if you pass in initial_solutions, you should not have to pass in x0 since x0 is only needed for initial solutions.